src/Graph.ts

```diff
@@ -523,6 +524,7 @@ export default class Graph {
 
 			for (let depModule of module.dependencies) {
 				if (depModule instanceof ExternalModule) continue;
+				if (depModule instanceof WasmModule) continue;
```

Comment:
Perhaps we should not permit `WasmModule` dependencies at all initially, and throw before even populating dependencies with a WasmModule?